pkg/operator/apiserver/audit/testdata/multipleCr.yaml

```diff
@@ -0,0 +1,71 @@
+apiVersion: audit.k8s.io/v1
+kind: Policy
+metadata:
+  name: policy
+# Don't generate audit events for all requests in RequestReceived stage.
+omitStages:
+  - "RequestReceived"
+rules:
+# Don't log requests for events
+- level: None
+  resources:
+  - group: ""
+    resources: ["events"]
+# Don't log authenticated requests to certain non-resource URL paths.
+- level: None
+  userGroups: ["system:authenticated", "system:unauthenticated"]
+  nonResourceURLs:
+  - "/api*" # Wildcard matching.
+  - "/version"
+  - "/healthz"
+  - "/readyz"
+# exclude resources where the body is security-sensitive
+- level: Metadata
+  resources:
+  - group: "route.openshift.io"
+    resources: ["routes"]
+  - resources: ["secrets"]
+  userGroups:
+  - system:authenticated:oauth
+- level: Metadata
+  resources:
+  - group: "oauth.openshift.io"
+    resources: ["oauthclients"]
+  userGroups:
+  - system:authenticated:oauth
+# log request and response payloads for all write requests
+- level: RequestResponse
+  verbs:
+  - update
+  - patch
+  - create
+  - delete
+  - deletecollection
+  userGroups:
+  - system:authenticated:oauth
+# catch-all rule to log all other requests at the Metadata level.
+- level: Metadata
+# Long-running requests like watches that fall under this rule will not
+# generate an audit event in RequestReceived.
+  omitStages:
+  - RequestReceived
+  userGroups:
+  - system:authenticated:oauth
+#- level: None
+- level: Metadata
+  resources:
+    - group: "route.openshift.io"
+      resources: ["routes"]
+    - resources: ["secrets"]
+  userGroups:
+    - system:authenticated
+- level: Metadata
+  resources:
+    - group: "oauth.openshift.io"
+      resources: ["oauthclients"]
+  userGroups:
+    - system:authenticated
+- level: RequestResponse
+  userGroups:
+    - system:authenticated
+- level: None
```

Comment:
nit: we should add a new line

Reply:
Balanced with this: https://github.com/openshift/library-go/pull/1126#discussion_r669383572